BookPlayer/Constants.swift

```diff
@@ -36,4 +36,8 @@ enum Constants {
         case normal = 1.0
         case boosted = 2.0
     }
+
+    enum Activities: String {
+        case playback = "com.tortugapower.audiobookplayer.activity.playback"
+    }
```

Comment:
Should this & the `info.plist` file info be dynamic to reflect the BundleID so we don't have a mix of identifiers for forked versions?

`case playback = "\(NSBundle.mainBundle().bundleIdentifier).activity.playback"`

Reply:
totally, thanks for pointing this out @ryantstone 👌

Reply:
It might be reasonable with the number of us working on the project to at some point in the near future move to a method of handling Bundle ID and possibly provisioning profiles to another setup so we're not accidentally including our provisioning profiles and custom ID's in PR's